platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1056,6 +1017,22 @@ public void performPcesRecovery() {
         }
     }
 
+    /**
+     * Offers the given state to the hash logger
+     *
+     * @param signedState the state to log
+     */
+    private void logSignedStateHash(@NonNull final SignedState signedState) {
```

Comment:
The advanced transformer does exactly this. Maybe leave a future work comment saying we should figure out a way to use it and get rid of this method.